### PR TITLE
Render background in ppu

### PR DESCRIPTION
**Pull request by johnor**

### CODE REVIEW

File: core/src/ppu.cpp [hunk 1]
```diff
@@ -455,6 +458,60 @@ void Ppu::fetch() {
...
+    // Each palette is 4 bytes and starts at 0x3F00.
+    const uint16_t color_address =
+            0x3F00u + combined_palette * 4u + combined_pixel;
+    // TODO(JN): Handle greyscale.
+    const uint8_t palette_index = mmu_->read_byte(color_address);
+    const auto color = get_color_from_palette_index(palette_index);
+
+    return Pixel{.x = static_cast<uint8_t>(cycle() - 1u),
+            .y = static_cast<uint8_t>(scanline()),
+            .color = color};
```
Comment (robinlinden) [inline core/src/ppu.cpp:512]: What did clang-format get up to here? :x Does adding newlines help anything, or is this really how clang-format wants it?

File: core/test/src/test_rom.cpp [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#include "ippu_helpers.h"
```
Comment (robinlinden) [inline core/test/src/test_rom.cpp:1]: Why is this required here all of a sudden?

**Review by robinlinden — APPROVED** [review]

File: core/src/ppu.cpp (hunk 1 above)
Comment (johnor) [reply core/src/ppu.cpp]: Adding newlines did not help. I created new variables for x and y to clean it up a bit.

File: core/test/src/test_rom.cpp (hunk 2 above)
Comment (johnor) [reply core/test/src/test_rom.cpp:1]: Needed in order for gtest to find PrintTo() and use the same definition everywhere.
